eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md

```diff
@@ -67,6 +67,21 @@ __Type:__ Integer
 __Required:__ Yes  
 __Description:__ The execution priority of the batch, as compared to other batches in the collection. A lower value SHALL be interpreted as a higher priority.
 
+#### data.batches.configuration
+__Type:__ Object[]
+__Required:__ No
+__Description:__ Any configuration required for all recipes. This is a custom field and is highly dependent on the test execution framework. Typically used for the set up of the execution space where the test cases can be executed. Example could be to override default configurations in the test execution framework. This property is a list of key-value pairs in the same format as [data.customData](../customization/custom-data.md). Note that this is a per-batch configuration and will apply to all recipes. Not to be confused with __data.batches.recipes.constraints__ which apply to only a single recipe.
```

Comment:
Again, as this parameter is the same as recipe constraints, the description should be more similar to that one. Also, please state the need to be careful if providing both batch level constraints and recipe level constraints to the same TERCC. I'd like to see that elaborate on the top of this page and then referenced from here as well as from the documentation for the recipe constraints.

Reply:
Changed documentation, please evaluate